src/modules/mc_pos_control/McPosControl.cpp

```diff
@@ -0,0 +1,1955 @@
+
+#include <px4_config.h>
+#include <px4_defines.h>
+#include <px4_tasks.h>
+#include <px4_posix.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <math.h>
+#include <arch/board/board.h>
+
+
+
+#include <float.h>
+#include <systemlib/mavlink_log.h>
+#include <platforms/px4_defines.h>
+
+#include "McPosControl.hpp"
```

Comment:
Maybe we should have a crude style guide. I think our naming and things like header extensions (.h or .hpp) are pretty inconsistent.